app/javascript/chat/__tests__/message.test.jsx

```diff
@@ -38,4 +39,22 @@ describe('<Message />', () => {
 
     expect(profileLink.parentElement).toHaveStyle({ color: msg.color });
   });
+
+  it('should highlight @mentions to the logged in user', () => {
```

Comment:
nitpick (non-blocking): Change the wording a bit.

```suggestion
  it('should highlight @mentions for the logged on user', () => {
```

Reply:
logged **on**??? 🧐